Sources/BraintreeDropIn/Custom Views/BTEnrollmentVerificationViewController.m

```diff
@@ -65,44 +69,33 @@ - (void)viewDidLoad {
     self.resendSmsButton = [UIButton new];
     self.resendSmsButton.translatesAutoresizingMaskIntoConstraints = NO;
     [self.resendSmsButton setTitle:smsButtonText forState:UIControlStateNormal];
-    
-    NSAttributedString *normalValidateButtonString = [[NSAttributedString alloc] initWithString:smsButtonText
-                                                                                     attributes:@{NSForegroundColorAttributeName:[BTUIKAppearance sharedInstance].tintColor,
-                                                                                                  NSFontAttributeName:[BTUIKAppearance sharedInstance].bodyFont}];
-    [self.resendSmsButton setAttributedTitle:normalValidateButtonString forState:UIControlStateNormal];
-    NSAttributedString *disabledValidateButtonString = [[NSAttributedString alloc] initWithString:smsButtonText
-                                                                                       attributes:@{NSForegroundColorAttributeName:[BTUIKAppearance sharedInstance].disabledColor,
-                                                                                                    NSFontAttributeName:[BTUIKAppearance sharedInstance].bodyFont}];
-    [self.resendSmsButton setAttributedTitle:disabledValidateButtonString forState:UIControlStateDisabled];
-    
+
+    [self.resendSmsButton setTitleColor:[BTUIKAppearance sharedInstance].tintColor forState:UIControlStateNormal];
+    [self.resendSmsButton setTitleColor:[BTUIKAppearance sharedInstance].disabledColor forState:UIControlStateDisabled];
+
+    self.resendSmsButton.titleLabel.font = [BTUIKAppearance sharedInstance].staticHeadlineFont;
```

Comment:
Can you remind me why this button font doesn't scale along with dynamic text?

Reply:
We've been giving the buttons a static font size throughout Drop-in because that matches what Apple seems to do with buttons. I'm not sure if this is a place where we'd want to break that rule, though, since we do have more space on the page? We can also make a note to run this by the accessibility team and see what they think.

Reply:
Yeah following what Apple does makes sense to me. Running it by the accessibility team would probably be a good idea, though.